test/framework/src/test/java/org/elasticsearch/search/internal/ConcurrentSearchTestPluginTests.java

```diff
@@ -40,7 +40,7 @@ public void testEagerConcurrentSearch() throws IOException {
     }
 
     @Override
-    protected boolean eagerConcurrentSearch() {
+    protected boolean enableConcurrentSearch() {
```

Comment:
nit: Maybe let's rename the internal variable too from eagerConcurrentSearch to concurrentSearch? The name of the test can be changed too.